app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  def self.update_comments_counter
+    post.increment!(:commentsCounter)
+  end
```

Comment:
- [ ] [OPTIONAL] Just like I recommended in the post counter above. I will also like to recommend that can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_like_counter method every time a new post has been saved to database, then make the update_comment_counter method private.